src/index.html

```diff
@@ -20,8 +21,348 @@
       href="styles/index.scss"
     />
   </head>
-
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__link-item">
+            <a
+              class="nav__link is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
```

Comment:
The `href` attribute of the anchor tag should have a valid URL. Here, '/apple' is not a valid URL. If you want to prevent the link from navigating to another page, you can use a '#' or 'javascript:void(0);'.